packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -121,110 +122,70 @@ export default function MongoNav({
     projects,
   } = data;
 
-  const accountHost = hosts?.account ?? `https://account.mongodb.com`;
-  const cloudHost = hosts?.cloud ?? `https://cloud.mongodb.com`;
-  const universityHost = hosts?.university ?? `https://university.mongodb.com`;
-  const supportHost = hosts?.support ?? `https://support.mongodb.com`;
-
-  const sanitizedHosts: Required<HostsInterface> = {
-    account: accountHost,
-    cloud: cloudHost,
-    university: universityHost,
-    support: supportHost,
-    charts: hosts?.charts ?? `https://charts.mongodb.com`,
-    stitch: hosts?.stitch ?? `https://stitch.mongodb.com`,
+  const defaultHosts: Required<HostsInterface> = {
+    account: 'https://account.mongodb.com',
+    cloud: 'https://cloud.mongodb.com',
+    university: 'https://university.mongodb.com',
+    support: 'https://support.mongodb.com',
+    charts: 'https://charts.mongodb.com',
+    stitch: 'https://stitch.mongodb.com',
   };
 
-  const defaultOrgURL = (orgId: string) =>
-    `${cloudHost}/v2#/org/${orgId}/projects`;
-  const constructOrganizationURL =
-    constructOrganizationURLProp ?? defaultOrgURL;
-
-  const defaultProjectURL = (projectId: string) =>
-    `${cloudHost}/v2#/${projectId}`;
-  const constructProjectURL = constructProjectURLProp ?? defaultProjectURL;
+  const constructedHosts = defaultsDeep(hosts, defaultHosts);
```

Comment:
optional: Perhaps rename the internal value of the prop so it's `({ hosts = hostsProps })` and then just name this variable `hosts`.

Ditto for URLs.

Reply:
👍